src/ioq_server2.erl

```diff
@@ -125,8 +125,11 @@ call_int(Fd, Msg, Dimensions, IOType) ->
 ioq_server(#ioq_request{}, search) ->
     ?IOQ2_SEARCH_SERVER;
 ioq_server(#ioq_request{fd=Fd}, _) ->
-    DispatchStrategy = config:get(
-        "ioq2", "dispatch_strategy", ?DISPATCH_SERVER_PER_SCHEDULER),
+    DispatchStrategy0 = config:get("ioq2", "dispatch_strategy", undefined),
+    DispatchStrategy = case DispatchStrategy0 of
+        undefined         -> ?DISPATCH_SERVER_PER_SCHEDULER;
+        DispatchStrategy0 -> list_to_atom(DispatchStrategy0)
+    end,
```

Comment:
Perhaps use a helper function, e.g `config_get_atom("ioq2", "dispatch_strategy", ?DISPATCH_SERVER_PER_SCHEDULER)`?

Reply:
I think it would be worthwhile to add a `config:get_atom/*` set of functions, but for the time being I moved this into `ioq_config:get_dispatch_strategy/0` in 57dd65f which cleaned it up.